code/datums/wires/suit_storage_unit.dm

```diff
@@ -36,6 +32,7 @@
 		if(WIRE_SAFETY)
 			S.safeties = !S.safeties
 		if(WIRE_SHOCK)
+			S.shock(usr, 50)
```

Comment:
`usr` isn't acceptable, this proc receives the `user` already as a parameter, adjust the proc signature and use that instead